### PR TITLE
Docs: snake_case queries

### PR DESCRIPTION
**Pull request by archiewood**
### Checklist

- [ ] For UI or styling changes, I have added a screenshot or gif showing before & after
- [ ] I have added a [changeset](https://github.com/evidence-dev/evidence/blob/main/CONTRIBUTING.md#adding-a-changeset)
- [x] I have added to the docs where applicable
- [ ] I have added to the [VS Code extension](https://github.com/evidence-dev/evidence-vscode) where applicable

### CODE REVIEW

File: sites/docs/pages/core-concepts/queries/index.md [hunk 1]
```diff
@@ -23,6 +23,14 @@ When you open a page in dev mode, Evidence runs all of the queries on the page.
 
 You include SQL queries in your page using a markdown code fence (starting and ending with 3 backticks). Evidence requires a query name to be supplied directly after the first 3 backticks.
 
+<Alert status=warning>
+
+**Use Snake Case for Queries**
+Use `snake_case` query names, query files and query aliases. Don't use `PascalCase`, `camelCase` or `names with-other/separators` - these may break in unexpected ways.
```
Comment (archiewood) [inline sites/docs/pages/core-concepts/queries/index.md:29]:
```suggestion

Use `snake_case` query names, query files and query aliases. Don't use `PascalCase`, `camelCase` or `names with-other/separators` - these may break in unexpected ways.
```

**Comment by zachstence** [comment]
My preference is `camelCase`, as that fits with the rest of the product (and JS as a whole) the best.

Also, I don't think the docs are correct in stating that `camelCase` or `PascalCase` would "break in unexpected ways". I'm not even sure that special characters would break anything

**Comment by archiewood** [comment]
Sorry I should have linked the issue here: https://github.com/evidence-dev/evidence/issues/2918

**Comment by zachstence** [comment]
> Sorry I should have linked the issue here: #2918
> 
> It specifically breaks when you alias queries with MixedCases - we then give the id mixedcases
> 
> I'm happy to:
> 
> * just add the warning about aliases which are the only part that do not allow other casings
> * not document this, but then we should change the behaviour of the aliasing, which currently lowercases any query ids you supply

If there's a good reason for forcing lowercase then I think this PR is good but we should also add a warning/error log if someone uses caps.

If there's not a good reason for forcing lowercase then I think we should undo that so any casing can be used.

@ItsMeBrianD any input here?

**Comment by archiewood** [comment]
> If there's not a good reason for forcing lowercase then I think we should undo that so any casing can be used.

Let's do this

**Pull request closed without merging**